core/libp2p/network/router.hpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_STREAM_MANAGER_HPP
+#define KAGOME_STREAM_MANAGER_HPP
+
+#include "libp2p/peer/protocol.hpp"
+#include "libp2p/stream/stream.hpp"
```

Comment:
You use both deprecated and new streams in this class. Please, stick to the new one in 62 line